Library/Homebrew/cask/cmd/info.rb

```diff
@@ -41,11 +41,12 @@ def self.get_info(cask)
         output << artifact_info(cask) + "\n"
         caveats = Installer.caveats(cask)
         output << caveats if caveats
-        output
+        puts output
+        Analytics.formula_output(cask)
```

Comment:
Could this method get renamed from `formula_output` (as this isn't a formula output here).

Reply:
done